lib/functions/NamedFunctions.ts

```diff
@@ -27,7 +25,7 @@ type Term = E.TermExpression;
 // https://www.w3.org/TR/xpath-functions/#casting-from-primitive-to-primitive
 // ----------------------------------------------------------------------------
 
-const toString = {
+const toStringNamed = {
```

Comment:
Why did you rename these?

Reply:
I don't really know, where but according to the linter this function already exists in the global-scope.